### ISSUE TITLE
Analogous to powsybl in Java , the substation should not be mandatory for creating buses.

### ISSUE THREAD

**Issue by AnkurArohi** [powsybl/pypowsybl#419]
* **Do you want to request a *feature* or report a *bug*?**
Feature

* **What is the current behavior?**
Currently one needs to set the substation and voltage level to be able to define voltage_level_id for the buses,this should not be a mandatory value but optional

* **If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem**

* **What is the expected behavior?**
Bus should be created without specifying voltage_level_id, this should be either done automatically or be only requested while drawing network line diagram/SLDs

* **What is the motivation / use case for changing the behavior?**
Usability Enhancement

* **Please tell us about your environment:**
  - PowSyBl Version: ...
  - OS Version: ...

* **Other information** (e.g. detailed explanation, stacktraces, related issues, suggestions how to fix, links for us to have context, eg. stackoverflow, spectrum, etc)

(if a question doesn't apply, you can delete it)

**Comment by sylvlecl** [comment]
Note that the java lib does not support creating buses without a containing voltage level:
only substations are optional !

So in pypowsybl we'll only be able to make substation optional as well, not the voltage level.
Really removing the requirement for a voltage level is not possible, because it does contain some necessary data that is not, today carried by the bus: in particular the nominal voltage.

However, of course, it's easy to wrap together the creation of a bus and a voltage level in a unique method of yours if you like, but I would not be in favor of putting it in the `Network` class, because it could lead the user to think that no voltage level is created, and that all data is available in buses dataframe, whereas this is not what will happen.

**Comment by AnkurArohi** [comment]
> Note that the java lib does not support creating buses without a containing voltage level: only substations are optional !
> 
> So in pypowsybl we'll only be able to make substation optional as well, not the voltage level. Really removing the requirement for a voltage level is not possible, because it does contain some necessary data that is not, today carried by the bus: in particular the nominal voltage.
> 
> However, of course, it's easy to wrap together the creation of a bus and a voltage level in a unique method of yours if you like, but I would not be in favor of putting it in the `Network` class, because it could lead the user to think that no voltage level is created, and that all data is available in buses dataframe, whereas this is not what will happen.

makes sense to me

**Comment by sylvlecl** [comment]
Implemented in #423: the substation will not be required any more for creating voltage levels:
```python
    net.create_voltage_levels(id='VL',
                              high_voltage_limit=250,
                              low_voltage_limit=200,
                              nominal_v=225,
                              topology_kind='BUS_BREAKER')
```
